uber/templates/registration/form.html

```diff
@@ -240,7 +240,16 @@
     </tr>
     <tr>
         <td> Signed Up </td>
-        <td> <i>{{ attendee.registered }}</i> </td>
```

Comment:
Another timezone fix, plus showing checked-in time, which I thought we were already doing.